apps/fluent-tester/src/E2E/ActivityIndicator/specs/ActivityIndicator.spec.win.ts

```diff
@@ -12,7 +12,6 @@ describe('Activity Indicator Testing Initialization', function () {
 
   it('Click and navigate to Activity Indicator test page', async () => {
     await ActivityIndicatorPageObject.scrollToComponentButton(Platform.Win32);
```

Comment:
We don't need this line anymore for Win32 specs.
`await ActivityIndicatorPageObject.scrollToComponentButton(Platform.Win32);`

It should be deleted just like the line below it

Reply:
done